models/hymod/include/Hymod.h

```diff
@@ -91,8 +91,19 @@ class hymod_kernel
         return 0.0;
     }
 
+    //! stub function to simulate losses due to evapotransportation
+    static double calc_et2(double soil_m, void* et_params)
```

Comment:
I think we should rename this function to calc_et_<descriptive et function name here>

Reply:
maybe calc_et_pdm03?